docs/managing_deis/upgrading-deis.rst

```diff
@@ -68,6 +68,77 @@ Upgrade Deis clients
 As well as upgrading ``deisctl``, make sure to upgrade the :ref:`deis client <install-client>` to
 match the new version of Deis.
 
+Graceful Upgrade
+----------------
+
+Alternatively, an experimental feature exists to provide the ability to perform a graceful upgrade. This process is
+available for version 1.9.0 moving foward and is intended to facilitate upgrades within a major version - upgrading
```

Comment:
"(for example, from v1.9.0 to v1.10.0)". Not everyone knows what semver major/minor/patch means.